tests/test_fieldset.py

```diff
@@ -346,6 +349,88 @@ def test_vector_fields(mode, swapUV):
         assert abs(pset[0].lat - .5) < 1e-9
 
 
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+@pytest.mark.parametrize('time_periodic', [4*86400.0, False])
+@pytest.mark.parametrize('cs', [False, 'auto', (1, 32, 32)])
```

Comment:
Call this variable `field_chunksize` instead of `cs` for readability?

Reply:
yes, we can do that